src/concrete/ml/pytest/utils.py

```diff
@@ -95,9 +97,25 @@
         id=get_model_name(model),
     )
     for model in _classifier_models
+    if get_model_name(model) != "KNeighborsClassifier"
     for n_classes in [2, 4]
+] + [
+    pytest.param(
```

Comment:
so, in all tests we check the KNN on a 3-dimensional, 10-sample dataset with 2 classes, is that correct ? 

Is 10 examples not too big for the CI ? We could do 2 examples no ?